cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/KafkaBrowser/KafkaBrowser.scss

```diff
@@ -21,18 +21,22 @@ $header-border: #cccccc;
 $header-bg: #eeeeee;
 $content-font-color: #333333;
 $hr_color: #999999;
+$dataprep-top-panel-height: 50px;
```

Comment:
nit: we already have declared in the `DataprepBrowserVariables.scss` right? Any specific reason we have redeclared it here?